person.rb

```diff
@@ -0,0 +1,25 @@
+# Person class
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
Unless needed, rather use keyword arguments for all the arguments as it makes the code read better.

Reply:
ok cool, will do this